build/_k-standard-goals.shade

```diff
@@ -227,6 +241,11 @@ default Quiet='${ false }'
     E("KOREBUILD_PARALLEL", "1");
    }
 
+#--verifynupkgs
```

Comment:
Let's just have the environment variable. We'll figure out a way to make it work via passed in arguments once @victorhurdugaci updates sake.